client/src/index.css

```
@@ -4,15 +4,15 @@
    =========================================== */
 
 /* 1. Core Variables & Base */
-@import './styles/_variables.css';
-@import './styles/base.css';
+@import url('./styles/_variables.css');
```

Stylelint (17.3.0)

[error] Expected "url('./styles/_variables.css')" to be "'./styles/_variables.css'" (import-notation)

(import-notation)